src/platform/packages/private/kbn-management/settings/application/hooks/use_solution_view.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { useState, useEffect } from 'react';
+import { SolutionView } from '@kbn/spaces-plugin/common';
+import { Space } from '@kbn/spaces-plugin/common';
+
+/**
+ * React hook which returns the solution view of the current active space.
+ */
+export function useSolutionView(getActiveSpace: () => Promise<Pick<Space, 'solution'>>) {
+  const [solutionView, setSolutionView] = useState<SolutionView>();
```

Comment:
nit: you could call `useServices` from here, to make it possible to call this function without a parameter:
```suggestion
export function useSolutionView() {
  const { getActiveSpace } = useServices();
  const [solutionView, setSolutionView] = useState<SolutionView>();
```

Reply:
Good call, thanks! Added with [2cb0e71](https://github.com/elastic/kibana/pull/209136/commits/2cb0e7176e0dcb5a5b8d700ae920e82a02c6470b).